### ISSUE TITLE
Update to 0.6.6 broken plotting

### ISSUE THREAD

**Issue by saboluigi** [guydavis/machinaris#448]
Updated to 0.6.6 on an Unraid server, causing the following error message when starting plotting.

Errors during plotman status:
 Traceback (most recent call last):
  File "/chia-blockchain/venv/bin/plotman", line 8, in 
    sys.exit(main())
  File "/chia-blockchain/venv/lib/python3.9/site-packages/plotman/plotman.py", line 247, in main
    with cfg.setup():
  File "/usr/lib/python3.9/contextlib.py", line 117, in __enter__
    return next(self.gen)
  File "/chia-blockchain/venv/lib/python3.9/site-packages/plotman/configuration.py", line 470, in setup
    plotman.plotters.madmax.check_configuration(
  File "/chia-blockchain/venv/lib/python3.9/site-packages/plotman/plotters/madmax.py", line 34, in check_configuration
    completed_process = subprocess.run(
  File "/usr/lib/python3.9/subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'chia_plot'

**Comment by guydavis** [comment]
Please see: https://github.com/guydavis/machinaris/wiki/Troubleshooting
Then attach the contents of `/tmp/madmax_setup.log` from within your container:
```
docker exec -it machinaris bash
cp /tmp/madmax_setup.log /root/.chia/
```
Find the madmax_setup.log in ~/.machinaris file and attach it here.  

As per the Github defect ticket template, please provide all details requested.  Thanks!

**Comment by guydavis** [comment]
Great!  Very interesting issue as it seems to only affect certain hardware. In the Discord, it seems this [proposed patch ](https://github.com/guydavis/machinaris/pull/447/files) from @karaliux.  I am currently investigating and testing.  Will update here when a fix that works for all is ready.

**Comment by cptnyehy0** [comment]
having the same issue and error. Would it be helpful for me to include my logs and details as well?

**Comment by guydavis** [comment]
Hi all, thanks for your patience on this one.  Big thanks to @karaliux for pointing out a solution for this.  As well, I see upstream Madmax repository just [got a commit](https://github.com/madMAx43v3r/chia-plotter/commit/c99a449c0467923c576acffcb1b7152ff94bb289) as well. I've included both fixes in the v0.6.7 `:test` images for Machinaris.

To confirm, please update your Chia (`machinaris`) and Chives (`machinaris-chives`) containers you plot with to the `:test` image (pre v0.6.7).  This will include the fix for building madmax.  After launch of the container, please allow at least 15 minutes for full startup.  Then try out plotting and let me know if you are able to plot again with Madmax.

**Comment by k2skaterii** [comment]
Never a fan of "It's working don't fix it"... And a fan of Sailor Jerry + Pepsi...  I was convinced that I screwed something up when I couldn't plot in 0.6.6.  

At the time I was plotting on my TrueNAS-Scale system - and I've been noticing 180GB of RAM sitting idle on that system.  Right around the time things stopped working, I was attempting to mount a RAM Disk to my container.  From there things spiraled into - DELETE the container.

If it's useful, I can attempt to recreate and provide logs.  Probably more useful if I can deploy the test image to my Scale system and let ya know though.

**Comment by cptnyehy0** [comment]
Test branch resolved my error and madmax plotting issue, thanks for the quick action!

**Comment by saboluigi** [comment]
Test branch also resolved mine, thank you so much!

**Comment by zybron** [comment]
In case you missed it in the discord, the test branch fixed my plotting issue as well.

**Comment by k2skaterii** [comment]
Missed it in discord.  I totally did.  Should be working right now... Entertaining a 4yr old... Trying to figure out how to modify this chart to pull the test image.  Didn't get to Discord ;)Thank you!On Nov 30, 2021 10:39 AM, zybron ***@***.***> wrote:
In case you missed it in the discord, the test branch fixed my plotting issue as well.

**Comment by guydavis** [comment]
Please try with now released [version 0.6.7](https://github.com/guydavis/machinaris/releases/tag/0.6.7).

**Comment by guydavis** [comment]
Marking as closed, now that version 0.6.7 is out.
